transport.go

```diff
@@ -207,10 +208,8 @@ func (d *defaultFetcher) fetchURI(uri string, queryType QueryType, queryValue st
 		return resp, ErrForbidden
 	}
 
-	contentType := resp.Header.Get("Content-Type")
-	contentTypeParts := strings.Split(contentType, ";")
-
-	if len(contentTypeParts) == 0 || contentTypeParts[0] != "application/rdap+json" {
+	contentType, _, err := mime.ParseMediaType(resp.Header.Get("Content-Type"))
```

Comment:
I think it would be nice to make the error message more specific by adding the received content type to it